com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -229,6 +244,25 @@ public void ConfigureInput(ScriptableRenderPassInput passInput)
             m_Input = passInput;
         }
 
+        public void ConfigureColorStoreAction(uint attachmentIndex, RenderBufferStoreAction storeAction)
```

Comment:
```suggestion
        public void ConfigureColorStoreAction(RenderBufferStoreAction storeAction, uint attachmentIndex = 0)
```

Suggestion to flip parameter order and then provide default parameter with 0. This would be more robust and allow easily to configure non-MRT case without caring about index. I think this API is also good enough that you don't need the get/setter properties above.

Reply:
sure, @jonuuukas raised the same suggestion. I preferred the explicit interface, but seems like I am outnumbered ;)
I'll change it to use 0 as default

Reply:
just for the record, I agree with @manuele-bonanno ;)

Reply:
Sure we can make it explicit if you prefer. No strong opinion. Can we then flip the order to the index is last?

Reply:
changed it to use the default value + stripped order, no strong preference regarding the default value